crates/biome_grit_patterns/src/grit_context.rs

```diff
@@ -282,7 +282,14 @@ fn file_owner_from_matches(
         return Ok(None);
     };
 
-    let absolute_path = name.absolutize()?.to_path_buf();
+    let base_dir = if cfg!(target_family = "wasm") {
+        PathBuf::from("/")
+    } else {
+        std::env::current_dir()?
```

Comment:
std::env::current_dir() was called inside `Absolutize::absolutize_from` implicity and caused "not supported on this platform" error.